resources/views/layouts/default.blade.php

```diff
@@ -480,6 +480,14 @@
                     @endcan
 
                   <li class="divider">&nbsp;</li>
+                    @can('checkin', \App\Models\Asset::class)
+                        <li{!! (Request::is('hardware/bulkcheckin') ? ' class="active"' : '') !!}>
+                            <a href="{{ route('hardware/bulkcheckin') }}">
+                                {{ trans('general.bulk_checkin') }}
+                            </a>
+                        </li>
+                    @endcan
+
```

Comment:
Putting the menu item above bulk checkout made the most sense to me. But again, feel free to offer a different suggestion.